benchmarks/dynamo/common.py

```diff
@@ -1948,7 +1948,9 @@ def init_optimizer(self, name, device, params):
             if (name in CI_USE_SGD and self.args.ci) or name in BENCHMARK_USE_SGD:
                 self.optimizer = torch.optim.SGD(params, lr=0.01, foreach=True)
             else:
-                self.optimizer = torch.optim.Adam(params, lr=0.01, foreach=True)
+                self.optimizer = torch.optim.Adam(
+                    params, lr=0.01, capturable=True, foreach=True
```

Comment:
```suggestion
                    params, capturable=True, foreach=True
```
Does the learning rate matter? The adam default is 0.001 btw

Reply:
Actually, what prevents us from just having `torch.optim.Adam()`? foreach should be auto specified and capturable should be enabled by compile, no?

Reply:
oh that's a good catch, maybe this will help with numeric stability. I'll update

Reply:
wait i still want an answer to my q tho ^